wolfhsm/wh_comm.h

```diff
@@ -120,8 +127,8 @@ typedef struct {
     uint16_t reqid;
     uint16_t seq;
     uint16_t size;
-    uint8_t packet[WOLFHSM_COMM_MTU];
-    whHeader* hdr;
+    uint8_t packet[WH_COMM_MTU];
+    whCommHeader* hdr;
     uint8_t* data;
     uint32_t client_id;
     uint32_t server_id;
```

Comment:
nitpick, but could we change the struct name to `whCommClientCtx` or `whCommClientContext` to make it explicit what the type is meant to represent (and be consistent with naming of our other structs)?

Reply:
Not a nit at all.  I'll correct in next push.